model/project_parser.go

```diff
@@ -1051,3 +1075,218 @@ func evaluateDependsOn(tse *tagSelectorEvaluator, tgse *tagSelectorEvaluator, vs
 	}
 	return newDeps, evalErrs
 }
+
+func (pp *ParserProject) mergeUnorderedUnique(toMerge *ParserProject, catcher grip.Catcher) {
```

Comment:
we should put comments at the top of each merge function, explaining what each merge rule is and what kind of things fall into it

Reply:
done